src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Geometry/BufferList/BufferList.xaml.cs

```diff
@@ -7,149 +7,240 @@
-        "")]
+        "This sample demonstrates how to use a planar (Euclidean) buffer operation by calling `GeometryEngine.Buffer` to generate one or more polygons from a collection of input geometries and a corresponding collection of buffer distances. The result polygons can be returned as individual geometries or unioned into a single polygon output.",
+        "Tap on the map to add points. Each point will use the buffer distance entered when it was created. The envelope shows the area where you can expect reasonable results for planar buffer operations with the North Central Texas State Plane spatial reference.",
+        "GeometryEngine, Geometry, Buffer, SpatialReference")]
     public partial class BufferList
     {
-        // Graphics overlay to display buffer related graphics.
-        private GraphicsOverlay _graphicsOverlay;
-
-        // List of geometry values (MapPoints in this case) that will be used by the GeometryEngine.Buffer operation.
-        private List<Geometry> _bufferPointsList = new List<Geometry>();
+        // A polygon that defines the valid area of the spatial reference used.
+        private Polygon _spatialReferenceArea;
 
-        // List of buffer distance values (in meters) that will be used by the GeometryEngine.Buffer operation.
-        private List<double> _bufferDistancesList = new List<double>();
+        // A Random object to create RGB color values.
+        private Random _random = new Random();
 
         public BufferList()
         {
             InitializeComponent();
 
-            // Create a map with a topographic basemap.
-            Map theMap = new Map(Basemap.CreateTopographic());
+            // Create the map and graphics overlays.
+            Initialize();
+        }
 
-            // Create an envelope that covers the Dallas/Fort Worth area.
-            Geometry startingEnvelope = new Envelope(-10863035.97, 3838021.34, -10744801.344, 3887145.299, SpatialReferences.WebMercator);
+        private void Initialize()
+        {
+            // Create a spatial reference that's suitable for creating planar buffers in north central Texas (State Plane).
+            SpatialReference statePlaneNorthCentralTexas = new SpatialReference(32038);
 
-            // Set the map's initial extent to the envelope.
-            theMap.InitialViewpoint = new Viewpoint(startingEnvelope);
+            // Define a polygon that represents the valid area of use for the spatial reference.
+            // This information is available at https://developers.arcgis.com/net/latest/wpf/guide/pdf/projected_coordinate_systems_rt100_3_0.pdf
```

Comment:
This is **awesome** - I had no idea we documented this and its super useful - will be great to get more eyeballs on it. :)